code/online_challenges/src/project_euler/problem_013/README.md

```diff
@@ -0,0 +1,116 @@
+# Project Euler Problem #13: Large Sum
+
+([Problem Link](https://projecteuler.net/problem=13))
+
+Work out the first ten digits of the sum of the following one-hundred 50-digit numbers.
+
+<p align = "center">
+37107287533902102798797998220837590246510135740250
```

Comment:
I would just put this in code formatting (It might be shorter + easier to read)

Reply:
I did thought of doing the same. But what about center alignment of these numbers in original question, I need to maintain? Should I add spaces for each number? Please guide @arnavb :thinking:

Reply:
@parulagg27 Don't worry about the center alignment; It's readability that matters the most!

Reply:
Completely agree. :+1: